doc/sphinx-guides/source/developers/dataset-migration-api.rst

```diff
@@ -5,7 +5,7 @@ The Dataverse software includes several ways to add Datasets originally created
 
 This experimental migration API offers an additional option with some potential advantages:
 
-* metadata can be specified using the json-ld format used in the OAI-ORE metadata export
+* metadata can be specified using the json-ld format used in the OAI-ORE metadata export (please note that the json-ld generated by OAI-ORE metadata export is not directly compatible with the Migration API, check example file below for reference)
```

Comment:
Can we say a little more about how the generated OAI-ORE format is not compatible? I'm a bit confused and not sure what I should be taking away from a look at the example file. 🤔